server/conf/application.conf

```diff
@@ -14,6 +14,11 @@ play.crypto.secret = "changeme"
 # ~~~~~
 play.i18n.langs = [ "en" ]
 
+jira.baseUrl = ${?JIRA_BASE}
+jira.endPoint_ticket = ${?JIRA_ENDPOINT_TICKET}
```

Comment:
I don't know why we need this to be configurable. The API endpoints are always static, and seems hardly unlikely that someone will deploy this app with a different endpoint to fetch issues.